src/Pages/Channels.razor

```diff
@@ -692,6 +715,11 @@
         return true;
     }
 
+    private bool OnWalletFilter(object itemValue, object searchValue)
+    {
+        if (itemValue == null) return false;
```

Comment:
Do you really need this line?

Reply:
yes, because on some cases you get null on the itemvalue